rust-nostd-esp/Dockerfile

```diff
@@ -1,16 +1,16 @@
-FROM espressif/idf-rust:all_1.64.0.0
+FROM espressif/idf-rust:all_latest
 
 USER esp
 ENV USER=esp
 
 # Install sccacahe
-RUN cargo install sccache
+RUN cargo install cargo-audit
```

Comment:
No need to install sccache? 

If this is the case, let's remove the comment above

Reply:
It is already included in the image that we are pulling: https://github.com/esp-rs/rust-build/blob/main/idf-rust.Containerfile#L31
Although this may change in the future (if we change it, I'll update the builders ofc)